README.md

```diff
@@ -1,12 +1,21 @@
+# Commercetools Bundles Starter (Custom Applications)
+The Commercetools Bundles starter project contains two Merchant Center custom applications for managing product bundles. 
+Product bundles are several goods or services that are sold to customers as a single combined package. Bundles are presented as distinct items during the shopping experience. They generally have a unique product display page and are offered through product listing pages or search results. Product bundles allow for increased customer value, shopping convenience and increased profit through larger average order value and opportunities for better inventory management.
 
-The static bundles solution handles the concept of putting together multiple related products for a total-sum price. An example static bundle is selling a camera, bag, and SD card for a single bundled price. The customer purchases a single bundle and then receives two or more distinct inventory items as selected by merchandising.  The bundles-static package contains the static bundles Merchant Center custom application and the platform-extension-static-bundles folder contains its backend extension.
+A traditional bundle involves removing the items from inventory, pre-packing the
+items as a group, and receiving the package under a newly created SKU. The items
+are no longer able to be purchased individually. This process creates a positive
+customer experience and simplifies fulfillment.
 
-Dynamic bundles allow for more complex bundling scenarios. Dynamic bundles specify a group of products where the user can select one product variant. Common use-cases include mix-&-match offers, where a user can select six craft beers to build a six-pack or build an entire outfit choosing a shirt, jacket, pants, and tie. Another common use-case is composite products, where the user can configure the final product, for example building a computer by choosing a CPU, GPU, RAM, and HDD.
-Where a static bundle points to specific SKUs, a dynamic bundle points to specific categories and stores business rules around each category. Pricing for the dynamic bundle can be a fixed amount or change based on the bundle selections. The bundles-dynamic package contains the dynamic bundles Merchant Center custom application and the backend-starter-dynamic-bundles contains starter resources for a dynamic backend.
+An alternative option is to create a new virtual SKU that represents the bundle.
+This new SKU points to all items in the bundle. After the customer purchases the
+virtual SKU, the items are packaged and delivered. This solution targets these
+virtual or post-pack bundles.
 
+Check detailed documentation about static and dynamic bundles projects,
+
+- [Bundles-static](docs/static/index.md)
+- [Bundles-dynamic](docs/dynamic/index.md)
```

Comment:
Why did you remove the parts about static and dynamic bundles?

Reply:
It was duplicated, Check in static and dynamic bundles docs for the removed sentences. 
So I have removed only the duplicated sentences.